demo/ssn-input.html

```diff
@@ -20,6 +20,10 @@
         display: inline-block;
       }
 
+      :host([hidden]) {
+        display: none!important;
```

Comment:
Sorry to be a nitpicker -- would you mind adding a space before the `!important`, please?